wp/lib/bap_wp/src/output.ml

```diff
@@ -41,86 +35,75 @@ let format_model (model : Model.model) (env1 : Env.t) (_env2 : Env.t) : string =
           (key, value)::pairs)
   in
   let fmt = Format.str_formatter in
-  List.iter key_val
-    ~f:(fun (var, value) ->
-        let var_str = Var.to_string var in
-        let pad_size = Int.max (5 - (String.length var_str)) 1 in
```

Comment:
My beautiful code!

Reply:
It's still there! I just moved it to `constraint.ml`, so I could reuse it to print out the registers in the path.